README.md

```diff
@@ -59,7 +59,7 @@ Once you've created your application in the [applications manager](https://devel
 need to edit the `./gradle.properties` file and change the
 `WP.OAUTH.APP_ID` and `WP.OAUTH.APP_SECRET` fields. Then you can compile and
 run the app on a device or an emulator and try to login with a WordPress.com
-account.
+account. Note that authenticating to WordPress.com via Google is not supported in development builds of the app, only in the offical release.
```

Comment:
Typo here - should be "official" rather than [offical](https://www.urbandictionary.com/define.php?term=Offical).

Reply:
Fixed!